azure-pipelines.yml

```diff
@@ -434,6 +439,86 @@ jobs:
     env:
       AZURE_CLI_TEST_TARGET_PROFILE: '2018-03-01'
 
+- job: RunAutomationReducedPython3_8
+  displayName: Run Automation Reduced Python 3.8
+  dependsOn: BuildPythonWheel
+  timeoutInMinutes: 90
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.8'
+    inputs:
+      versionSpec: 3.8
+  - task: DownloadPipelineArtifact@1
+    displayName: 'Download Python Wheels'
+    inputs:
+      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
+      artifactName: pypi
+  - bash: |
+      pip install --force-reinstall wheel
+  - task: Bash@3
+    displayName: 'Run Test Automation'
+    inputs:
+      targetType: 'filePath'
+      filePath: ./scripts/ci/test_automation.sh
+    env:
+      REDUCE_SDK: 'True'
+
+- job: RunAutomationReduced20190301Python3_8
+  displayName: Run Automation Reduced Python 3.8, Profile 2019-03-01
+  dependsOn: BuildPythonWheel
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.8'
+    inputs:
+      versionSpec: 3.8
+  - task: DownloadPipelineArtifact@1
+    displayName: 'Download Python Wheels'
+    inputs:
+      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
+      artifactName: pypi
+  - bash: |
+      pip install --force-reinstall wheel
+  - task: Bash@3
+    displayName: 'Run Test Automation'
+    inputs:
+      targetType: 'filePath'
+      filePath: ./scripts/ci/test_automation.sh
+    env:
+      REDUCE_SDK: 'True'
+      AZURE_CLI_TEST_TARGET_PROFILE: '2019-03-01'
+
+- job: RunAutomation20180301Python3_8
+  displayName: Run Automation Python 3.8, Profile 2018-03-01
+  dependsOn: BuildPythonWheel
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.8'
+    inputs:
+      versionSpec: 3.8
+  - task: DownloadPipelineArtifact@1
+    displayName: 'Download Python Wheels'
+    inputs:
+      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
+      artifactName: pypi
+  - bash: |
+      pip install --force-reinstall wheel
+  - task: Bash@3
+    displayName: 'Run Test Automation'
+    inputs:
+      targetType: 'filePath'
+      filePath: ./scripts/ci/test_automation.sh
```

Comment:
Actually I also have a question for test_automation.sh file. Typically we run test based on `azdev test`, but in ADO CI system, tests are run based on python https://github.com/Azure/azure-cli/blob/dev/scripts/ci/test_automation.sh#L57.

Reply:
Another good question to look into, this could be a discrepancy in developing and ci testing, @haroldrandom

Reply:
Yeah. It's a TODO for CI